modules/assocfile/assocfile.js

```diff
@@ -0,0 +1,74 @@
+//This module is "assocfile" instead of "associatefile", becase "associate" module is catching all commands starting with "associate"
+
+exports.help = function(commandPrefix) {
+	return [[commandPrefix + 'assocfile "<hook>" "<file_name>" or clear', 'Associate and disassociate a file with a phrase.', 'Associate and disassociate a file like image or video (from ~\files directory) with a phrase (so that a file is sent on a given phrase in chat).']];
+};
+
+exports.match = function(text, commandPrefix, thread, senderName, api) {
+	if (text.startsWith(commandPrefix + 'assocfile')) {
+		return true;
+	}
+
+	var s = text.toLowerCase();
+	for (var assoc in this.config[thread]) {
+		if (s.indexOf(assoc.toLowerCase()) !== -1) {
+			if (api) {
+				api.sendFile("file", "files\\"+this.config[thread][assoc], "", thread);
+			}
+			return true;
+		}
+	}
+	return false;
+};
+
+exports.toggleAssociation = function(thread, hook, text) {
```

Comment:
Nothing other than `help`, `run` and `match` should be exported. In a soon to be merged branch, the `associate